support/privacy-resources.md

```diff
@@ -0,0 +1,108 @@
+
+- [Prebid.js support for GPP](/dev-docs/modules/consentManagementGpp.html)
+- [Prebid Support for MSPA](/features/mspa-usnat.html).
+
+### Europe
+
+The privacy tools that Prebid has built in support of European rules may help address the requirements of the GDPR and the Digital Services Act.
+
+The IAB defined the Transparency and Consent Framework (TCF) to address European GDPR rules. Prebid support for TCF is described:
+
+- [Prebid.js CMP Best Practices](/dev-docs/cmp-best-practices.html)
+- [Prebid.js GDPR Consent Management Module](/dev-docs/modules/consentManagement.html)
+- [Prebid.js GDPR Enforcement Module](/dev-docs/modules/gdprEnforcement.html)
+- [Prebid Server GDPR Support](/prebid-server/features/pbs-privacy.html#gdpr)
+- [White paper: Prebid Support for Enforcing TCF 2](https://docs.google.com/document/d/1fBRaodKifv1pYsWY3ia-9K96VHUjd8kKvxZlOsozm8E/edit)
+
+### Canada
+
+Please see [Prebid's support for Quebec privacy law 25](/features/ac-quebec.html).
+
+## Global Privacy References
+
+### Chrome Privacy Sandbox
+
+Privacy Sandbox is the name the Chrome browser has given to a series of features aimed at smoothing the transition off the 3rd party cookie.
+
+#### Topics
+
+At a high level, the "Topics" feature is Chrome's way of defining a taxonomy of information that can be used for ad targeting. See 
+[Chrome Topics](https://privacysandbox.com/proposals/topics/) for details.
+
+There's actually nothing to do to enable Topics in Prebid – bidders will receive their own Topics if they've implemented that feature.
+That said, Prebid does have a [Topics FPD Module](/dev-docs/modules/topicsFpdModule.html) that allows bidders to share each other's Topics.
+
+#### Protected Audience API
+
+PAAPI is Chrome's solution for ad targeting done in a privacy-friendly way. In short, GAM will kick off an in-browser auction after
+the contextual auction. If the in-browser auction wins, it can override the ad chosen by GAM. 
+
+See [Chrome's PAAPI documentation](https://developers.google.com/privacy-sandbox/relevance/protected-audience) for the full background.
+
+To enable Interest Group bidding in Prebid, you can add the Prebid [Fledge For GPT Module](/dev-docs/modules/fledgeForGpt.html).
+
+##### Test Period
+
+During the first part of 2024, Chrome and GAM are running a test of PAAPI on a limited subset of traffic. However, the
+[Fledge For GPT Module](/dev-docs/modules/fledgeForGpt.html) enables Interest Group auctions 100% of the time. During the test
+period, publishers can better align browser and programmatic ad behavior by only enabling Interest Group auctions for
+the relevant Chrome testing labels. Here's example code:
+
+```javascript
+Promise.resolve(navigator.cookieDeprecationLabel?.getValue?.()).then(label => {
+    pbjs.setConfig({
+        fledgeForGpt: {
+            enabled: !label || label.startsWith("treatment_") || label === 'label_only_5'
```

Comment:
label 2-4 should also be included https://support.google.com/admanager/answer/13178817?hl=en

Reply:
`enabled: !label || label.startsWith("treatment_") || label1= 'label_only_1'` perhaps

Reply:
label 2-4 is treated the same as Chrome with no label 

Perhaps we offer
 enabled: label.startsWith("treatment_") || label === 'label_only_5'
and 
 enabled: !label || label.startsWith("treatment_") || label != 'label_only_1'
as two reasonable alternatives, but the example doesn't make sense to me

Reply:
Made an update. Getting closer?

Reply:
Does this look better @patmmccann ?